charts/jaeger/README.md

```diff
@@ -4,39 +4,64 @@
 
 ## Introduction
 
-This chart adds all components required to run Jaeger as described in the [jaeger-kubernetes](https://github.com/jaegertracing/jaeger-kubernetes) GitHub page for a production-like deployment. The chart default will deploy a new Cassandra cluster (using the [cassandra chart](https://github.com/kubernetes/charts/tree/master/incubator/cassandra)), but also supports using an existing Cassandra cluster, deploying a new ElasticSearch cluster (using the [elasticsearch chart](https://github.com/kubernetes/charts/tree/master/incubator/elasticsearch)), or connecting to an existing ElasticSearch cluster. Once the back storage available, the chart will deploy jaeger-agent as a DaemonSet and deploy the jaeger-collector and jaeger-query components as standard individual deployments.
+This chart adds all components required to run Jaeger as described in the [jaeger-kubernetes](https://github.com/jaegertracing/jaeger-kubernetes) GitHub page for a production-like deployment. The chart default will deploy a new Cassandra cluster (using the [cassandra chart](https://github.com/kubernetes/charts/tree/master/incubator/cassandra)), but also supports using an existing Cassandra cluster, deploying a new ElasticSearch cluster (using the [elasticsearch chart](https://github.com/elastic/helm-charts/tree/master/elasticsearch)), or connecting to an existing ElasticSearch cluster. Once the storage backend is available, the chart will deploy jaeger-agent as a DaemonSet and deploy the jaeger-collector and jaeger-query components as Deployments.
 
 ## Installing the Chart
 
 Add the Jaeger Tracing Helm repository:
 
 ```bash
-$ helm repo add jaegertracing https://jaegertracing.github.io/helm-charts
+helm repo add jaegertracing https://jaegertracing.github.io/helm-charts
 ```
+### Default
 
 To install the chart with the release name `jaeger`, run the following command:
 
 ```bash
-$ helm install jaeger jaegertracing/jaeger
+helm install jaeger jaegertracing/jaeger
 ```
 
-After a few minutes, you should see a 3 node Cassandra instance, a Jaeger DaemonSet, a Jaeger Collector, and a Jaeger Query (UI) pod deployed into your Kubernetes cluster.
+By default, the chart deploys the following:
+
+ - Cassandra statefulset
+ - Jaeger Agent DaemonSet
+ - Jaeger Collector Deployment
+ - Jaeger Query (UI) Deployment
+
+![Jaeger with Default components](images/jaeger-default.png)
+
+> Image represents agent as a sidecar and for more info to refer to [this](https://medium.com/jaegertracing/deployment-strategies-for-the-jaeger-agent-1d6f91796d09)
```

Comment:
```suggestion
> Please note: The above image shows the agent collocated with the app container as a sidecar, rather than as a separate DaemonSet pod. For more info about these two approaches please refer to [this article by @jpkrohling](https://medium.com/jaegertracing/deployment-strategies-for-the-jaeger-agent-1d6f91796d09).
```